clients/fides-js/src/lib/preferences.ts

```diff
@@ -253,6 +275,11 @@ export const updateConsent = async (
     validation?: UpdateConsentValidation;
   },
   consentMethod: ConsentMethod = ConsentMethod.SCRIPT,
+  eventExtraDetails: FidesEventExtraDetails = {
+    trigger: {
+      origin: "external",
+    },
+  },
```

Comment:
since the default consent method here is "script" we're also setting the default origin to "external" to match.